pkg/scheduler/api/resource_info.go

```diff
@@ -296,6 +296,38 @@ func (r *Resource) Multi(ratio float64) *Resource {
 	return r
 }
 
+// ScaleResourcesWithRatios multiples the resource with ratio provided for CPU, Memory, and ScalarResources
+func (r *Resource) ScaleResourcesWithRatios(ratio map[string]float64, defaultRatio float64) *Resource {
+	if ratio == nil {
+		return r
+	}
+	// This value is defined in the overcommit plugin
+	generalFactorKey := "overcommit-factor"
```

Comment:
Why depend on it?  ratio map just need key like cpu/mem, etc.

Reply:
fixex